packages/dashboard/src/components/queryEditor/helpers/resourceExplorerErrorState.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
the folder name seems to be not super appropriate... these is a component... helpers seems like utility functions.... probably just components might make sense ?